Mathlib/Algebra/Algebra/Subalgebra/Operations.lean

```diff
@@ -0,0 +1,79 @@
+/-
+Copyright (c) 2018 Kenny Lau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang, Antoine Chambert-Loir
+-/
+import Mathlib.Algebra.Algebra.Subalgebra.Basic
+import Mathlib.RingTheory.Ideal.Operations
+
+#align_import algebra.algebra.subalgebra.basic from "leanprover-community/mathlib"@"b915e9392ecb2a861e1e766f0e1df6ac481188ca"
+
+/-!
+# More operations on subalgebras
+
+In this file we relate the definitions in `RingTheory.Ideal.Operations` to subalgebras.
+The contents of this file are somewhat random since both `Subalgebra.Basic` and `Ideal.Operations`
+are somewhat of a grab-bag of definitions, and this is whatever ends up in the intersection.
```

Comment:
```suggestion
In this file we relate the definitions in `Mathlib.RingTheory.Ideal.Operations` to subalgebras.
The contents of this file are somewhat random since both `Mathlib.Algebra.Algebra.Subalgebra.Basic`
and `Mathlib.RingTheory.Ideal.Operations` are somewhat of a grab-bag of definitions, and this is
whatever ends up in the intersection.
```
(This should make them links)